pkg/reconciler/common/tektoninstallerset/types.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2022 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tektoninstallerset
+
+import (
+	"context"
+	"fmt"
+
+	mf "github.com/manifestival/manifestival"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type ComponentInstaller interface {
+	GetManifest(ctx context.Context) (*mf.Manifest, error)
+	GetLabels(ctx context.Context) map[string]string
+	GetAnnotations(ctx context.Context) map[string]string
+	GetOwnerReferences(ctx context.Context) *metav1.OwnerReference
```

Comment:
```suggestion
	GetOwnerReferences(ctx context.Context) []metav1.OwnerReference
```
let us make this change right away. 
as the ownerReference field on k8s resources is an array anyway.